frontend/src/storybook/StoryWrapper.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { BrowserRouter } from "react-router-dom";
+import {
+  useDatasetStoreProvider,
+  DatasetProvider,
+  startMetadataLoad,
+} from "../data/useDatasetStore";
+import Logger from "../utils/Logger";
+import { initGlobals } from "../utils/globals";
+import DataFetcher from "../data/DataFetcher";
+import CssBaseline from "@material-ui/core/CssBaseline";
+import { ThemeProvider } from "@material-ui/styles";
+import MaterialTheme from "../styles/MaterialTheme";
+
+export const StoryWrapper = (storyFn: any) => {
+  initGlobals(new Logger(false), new DataFetcher());
+  startMetadataLoad();
```

Comment:
I'd move the initGlobals and startMetadataLoad calls outside of the component so they aren't repeatedly called. (I should probably update the initGlobals method to throw an exception if it's called multiple times actually...)

Reply:
oh good call!